tests/storage/imageupload.go

```diff
@@ -28,13 +28,16 @@ import (
 	"kubevirt.io/client-go/kubecli"
 	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
 
```

Comment:
nit: newline

Reply:
Done

Reply:
Ok this newline was added by `make && make generate`.
I re-arranged all the imports with some logic :)